openpype/modules/sync_server/sync_server.py

```diff
@@ -229,6 +240,172 @@ def _get_configured_sites_from_setting(module, project_name, project_setting):
     return configured_sites
 
 
+def get_last_published_workfile_path(
+    host_name: str,
+    project_name: str,
+    task_name: str,
+    workfile_representation: dict,
+    anatomy: Anatomy = None,
+) -> str:
+    """Returns last published workfile path.
+    Args:
+        host_name (str): Host name.
+        project_name (str): Project name.
+        task_name (str): Task name.
+        workfile_representation (dict): Workfile representation.
+        anatomy (Anatomy, optional): Anatomy. Defaults to None.
+    Returns:
+        str: Last published workfile path.
+    """
+    if not anatomy:
+        anatomy = Anatomy(project_name)
+
+    if not workfile_representation:
+        print(
+            "No published workfilefor task '{}' and host '{}'.".format(
+                task_name, host_name
+            )
+        )
+        return
+
+    return get_representation_path_with_anatomy(
+        workfile_representation, anatomy
+    )
+
+
+def download_last_published_workfile(
+    host_name: str,
+    project_name: str,
+    asset_name: str,
+    task_name: str,
+    last_published_workfile_path: str,
+    workfile_representation: dict,
+    subset_id: str,
+    last_version_doc: dict,
+    anatomy: Anatomy = None,
+    asset_doc: dict = None,
+) -> str:
+    """Downloads the last pusblished workfile, and returns its path.
```

Comment:
Imperative and docstring formatting space and:
```suggestion
    """Download the last published workfile, and return its path.
```